src/julia-syntax.scm

```diff
@@ -2187,6 +2231,49 @@
            lhss)
        (unnecessary ,xx))))
 
+(define (destructure- i lhss xx n st end)
```

Comment:
Could use a comment explaining the arguments.

Reply:
Ok, will do